src/babel-preset-env.js

```diff
@@ -0,0 +1,18 @@
+import {
+  registerPreset,
+  transform as babelTransform,
+} from 'babel-standalone';
+
+registerPreset('env', require('./build-babel-preset-env.js'));
+
+export function transform (code, options = {}) {
```

Comment:
Do you think this function is beneficial? People can just call `Babel.transform` directly and just pass `env` as the preset. I don't think we need another separate entry point just for `babel-preset-env`. Babili is special since it's available as a separate CLI app.